agentstack/deploy.py

```diff
@@ -0,0 +1,94 @@
+import os
+import tempfile
+import tomllib
+import webbrowser
+import zipfile
+from pathlib import Path
+
+from agentstack.auth import get_stored_token, login
+from agentstack.conf import ConfigFile
+from agentstack.utils import term_color
+import requests
+
+
+def deploy():
+    bearer_token = get_stored_token()
```

Comment:
add a deploy stub to /cli that manages the CLI interaction and logging, then use the deploy function from this file